src/main/java/dk/cngroup/lentils/service/FinalPlaceService.java

```diff
@@ -36,6 +42,34 @@ public FinalPlace getFinalPlace() {
         throw new MoreFinalPlacesException("More final places found.");
     }
 
+    public void checkFutureOpeningTime(final FinalPlace finalPlace, final BindingResult bindingResult) {
+        if (!isOpeningTimeInFuture(finalPlace)) {
+            createOpeningTimeFieldError(finalPlace, bindingResult, TIME_IN_PAST_ERROR);
+        }
+    }
+
+    public Boolean isOpeningTimeInFuture(final FinalPlace finalPlace) {
+        try {
+            return finalPlace.getOpeningTime().isAfter(LocalDateTime.now());
+        } catch (Exception e) {
+            return true;
+        }
+    }
+
+    private void createOpeningTimeFieldError(final FinalPlace finalPlace,
+                                             final BindingResult bindingResult,
+                                             final String error) {
+        FieldError timeInPastError = new FieldError(
```

Comment:
Zkus mrknout na tohle https://docs.oracle.com/javaee/7/api/javax/validation/constraints/Future.html.
Není to to, co tady potřebuješ?

Reply:
Nebo tady link na novější verzi https://javaee.github.io/javaee-spec/javadocs/javax/validation/constraints/Future.html

Reply:
validace na cas v minuosti odstranena